### DISCUSSION TITLE
Format Catalog Member Item Name

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#6406] [Q&A]
Does anyone know if there is anyway to format the Catalog member items names?
I.E. Can you put in newlines or make text bold?

Thanks very much,
Lisa

**Comment by nf-s** [comment]
Hi Lisa

Sorry this isn't possible at the moment.
I'll create an issue for it!

https://github.com/TerriaJS/terriajs/issues/6413

Cheers,
Nick
